src/main/resources/lang/english.lang

```diff
@@ -83,9 +83,9 @@ skript command:
 	invalid script: Can't find the script <grey>'<gold>%s<grey>'<red> in the scripts folder!
 	invalid folder: Can't find the folder <grey>'<gold>%s<grey>'<red> in the scripts folder!
 	reload:
-		reloading: Reloading %s...
-		reloaded: Successfully reloaded %s.
-		error: Encountered %2$s error¦¦s¦ while reloading %1$s!
+		reloading: Reloading <gold>%s<reset>...
+		reloaded: <lime>Successfully reloaded <gold>%s<lime>. <gray>(<gold><bold>%2$sms<gray>)
```

Comment:
I don't think the reload time should be bold, but that is up for discussion :D

Reply:
Just for a formatting look, I like to use styling between bold and regular font-weight. Since it is changeable users are free to change it as they like :)